### PR TITLE
Update blur.cpp

### PR DESCRIPTION
**Pull request by wstaelens**
please verify, code optimizations.

### CODE REVIEW

**Comment by morevnaproject** [comment]
Hello @wstaelens ! Thank you for the contribution, I will forward this to our devs.

BTW, I've got a email notification that you sent me some messages via LinkedIn. Unfortunately, as I am residing in Russia, I cannot access LinkedIn. You can PM me via https://morevnaproject.org/about/contact/.

File: synfig-core/src/synfig/blur.cpp [hunk 1]
```diff
@@ -358,19 +361,21 @@ static void GaussianBlur_nxn(etl::surface<T,AT,VP> &surface,int n)
...
 			{
-				Tmp2=SC[i*2][x]+Tmp1;
-				SC[i*2][x]=Tmp1;
-				Tmp1=SC[i*2+1][x]+Tmp2;
-				SC[i*2+1][x]=Tmp2;
+				int idouble = i*2;
+				Tmp2=SC[idouble][x]+Tmp1;
+				SC[idouble][x]=Tmp1;
+				Tmp1=SC[idouble][x]+Tmp2;
+				SC[idouble+1][x]=Tmp2;
```
Comment (blackwarthog) [inline synfig-core/src/synfig/blur.cpp]: Tmp1=SC[idouble **+1** ][x]+Tmp2;
SC[idouble+1][x]=Tmp2;

File: synfig-core/src/synfig/blur.cpp [hunk 2]
```diff
@@ -500,9 +510,10 @@ bool Blur::operator()(const Surface &surface,
 		for(x=0;x<w;x++)
 		{
 			Color a = surface[y][x];
-			a.set_r(a.get_r()*a.get_a());
-			a.set_g(a.get_g()*a.get_a());
-			a.set_b(a.get_b()*a.get_a());
+			int aa = a.get_a();
+			a.set_r(a.get_r()*aa);
+			a.set_g(a.get_g()*aa);
+			a.set_b(a.get_b()*aa);
```
Comment (blackwarthog) [inline synfig-core/src/synfig/blur.cpp]: alpha is a ColorReal type (actually float), not int

**Comment by wstaelens** [comment]
ok, how does this code review work here on Git? Do i need to change anything or is this already been done by @blackwarthog ?

**Comment by wstaelens** [comment]
attempt to update the code review

**Comment by morevnaproject** [comment]
@blackwarthog Your opinion?

**Comment by ice0** [comment]
Thank you for your cooperation!

**Pull request merged**